packages/client/src/DataCommonsClient.ts

```diff
@@ -0,0 +1,480 @@
+    return (await (await fetch(url)).json()) as ApiNodePropvalOutResponse;
+  }
+
+  /**
+   * Fetches place/entity names
+   * @param entityDcids list of entity DCIDs
+   * @param prop optional entity name property
+   */
+  async getPlaceNames(params: {
+    dcids: string[];
+    prop?: string;
+  }): Promise<Record<string, string>> {
+    if (!params.dcids.length) {
+      return Promise.resolve({});
+    }
+    const url = `${this.apiRoot || ""}/api/place/name`;
+    const response = await fetch(url, {
+      method: "post",
+      body: JSON.stringify({
+        dcids: params.dcids,
+        prop: params.prop,
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    return (await response.json()) as Record<string, string>;
+  }
+
+  /**
+   * Gets and processes the data from /api/observations/point endpoint
+   * @param apiRoot api root
+   * @param entities list of entitites to get data for
+   * @param variables list of variables to get data for
+   * @param date date to get the data for
+   * @param alignedVariables groups of variables that should have the same unit
+   */
+  async getObservationsPoint(params: {
+    date?: string;
+    entities: string[];
+    variables: string[];
+  }): Promise<PointApiResponse> {
+    const queryString = toURLSearchParams({
+      date: params.date,
+      entities: params.entities,
+      variables: params.variables,
+    });
+    const url = `${this.apiRoot || ""}/api/observations/point?${queryString}`;
+    return (await (await fetch(url)).json()) as PointApiResponse;
+  }
+
+  /**
+   * Fetches point observations related to entities of type childType
+   * that are contained within the specified parentEntity
+   *
+   * Uses /api/observations/point/within endpoint
+   * @param childType type of entity to get data for
+   * @param parentEntity the parent entity of the entities to get data for
+   * @param variables list of variables to get data for
+   * @param date date to get the data for
+   */
+  async getObservationsPointWithin(params: {
+    parentEntity: string;
+    childType: string;
+    variables: string[];
+    date?: string;
+    facetIds?: string[];
+  }): Promise<PointApiResponse> {
+    const queryString = toURLSearchParams({
+      parentEntity: params.parentEntity,
+      childType: params.childType,
+      variables: params.variables,
+      date: params.date,
+      facetIds: params.facetIds,
+    });
+    const url = `${
+      this.apiRoot || ""
+    }/api/observations/point/within?${queryString}`;
+    return (await (await fetch(url)).json()) as PointApiResponse;
+  }
+
+  /**
+   * Gets the data from /api/observations/series endpoint.
+   * @param entities list of enitites to get data for
+   * @param variables list of variables to get data for
+   */
+  async getObservationsSeries(params: {
+    entities: string[];
+    variables: string[];
+    facetIds?: string[];
```

Comment:
include this in the list of params in the comment doc? & check the comments for other functions as well

Reply:
Removed facetIds entirely for now (revisiting in a future PR)